osu.Game.Rulesets.Osu/Skinning/Argon/OsuArgonSkinTransformer.cs

```diff
@@ -36,6 +36,9 @@ public OsuArgonSkinTransformer(ISkin skin)
                     }
 
                 case OsuSkinComponentLookup osuComponent:
+                    if (base.GetDrawableComponent(lookup) is Drawable c)
+                        return c;
```

Comment:
What's this doing?

Reply:
This is allowing `Skin.cs` to provide the implementation from disk.

Reply:
I'm not gonna lie, this type of change makes me a bit nervous because I don't feel like I have anything close to a good-enough grasp on the whole transformer sprawl to be able to tell if this is going to behave fine or not. At least superficially it _looks_ like something that may break a whole lot of nonsense...

Not sure. Maybe it would help if you can provide an explanation why this is fine and is in fact not going to break things.

Reply:
Truthfully, I don't think anyone has a good grasp on skinning as a whole right now, me included. I can't give you an explanation because it _could_ break things under the right circumstances.

This implementation also doesn't handle judgement pieces correctly, but I'm not sure how to actually do that right now because `Skin` defines an implementation for `GameplaySkinComponentLookup<HitResult>`...

Maybe there's a future where this method is split into two like:
- `LoadCustomDrawableComponent()` - loads only from the .json files.
- `LoadDefaultDrawableComponent()` - loads only from the skin-specific components (e.g. argon components).